### ISSUE TITLE
CI setup have started failing

### ISSUE THREAD

**Issue by hegjon** [python/typing_extensions#99]
Most likely due to `ubuntu-latest` changed from 20.04 to 22.04.

### Last success tests run 135:

> Operating System
>   Ubuntu
>   20.04
>   LTS

https://github.com/python/typing_extensions/actions/runs/3449085893/jobs/5756701950#step:1:4

### Non lint failed test run:

> Operating System
>   Ubuntu
>   22.04
>   LTS
> 

https://github.com/python/typing_extensions/actions/runs/3543481555/jobs/5955553976#step:1:4

**Comment by hegjon** [comment]
### Version available for 22.04
```
❯ jq '.[] | select (.files[].platform_version == "22.04") | .version' < versions-manifest.json
"3.12.0-alpha.2"
"3.12.0-alpha.1"
"3.11.0"
"3.11.0-rc.2"
"3.11.0-rc.1"
"3.11.0-beta.5"
"3.11.0-beta.4"
"3.11.0-beta.3"
"3.11.0-beta.2"
"3.11.0-beta.1"
"3.10.8"
"3.10.7"
"3.10.6"
"3.10.5"
"3.10.4"
"3.9.15"
"3.9.14"
"3.9.13"
"3.9.12"
"3.8.14"
"3.8.13"
"3.8.12"
"3.7.15"
"3.7.14"
"3.7.13"
```

### Version available for 20.04
```
❯ jq '.[] | select (.files[].platform_version == "20.04") | .version' < versions-manifest.json
"3.12.0-alpha.2"
"3.12.0-alpha.1"
"3.11.0"
"3.11.0-rc.2"
"3.11.0-rc.1"
"3.11.0-beta.5"
"3.11.0-beta.4"
"3.11.0-beta.3"
"3.11.0-beta.2"
"3.11.0-beta.1"
"3.11.0-alpha.7"
"3.11.0-alpha.6"
"3.11.0-alpha.5"
"3.11.0-alpha.4"
"3.11.0-alpha.3"
"3.11.0-alpha.2"
"3.11.0-alpha.1"
"3.10.8"
"3.10.7"
"3.10.6"
"3.10.5"
"3.10.4"
"3.10.3"
"3.10.2"
"3.10.1"
"3.10.0"
"3.10.0-rc.2"
"3.10.0-rc.1"
"3.10.0-beta.4"
"3.10.0-beta.3"
"3.10.0-beta.2"
"3.10.0-beta.1"
"3.10.0-alpha.7"
"3.10.0-alpha.6"
"3.10.0-alpha.5"
"3.10.0-alpha.4"
"3.10.0-alpha.3"
"3.10.0-alpha.2"
"3.10.0-alpha.1"
"3.9.15"
"3.9.14"
"3.9.13"
"3.9.12"
"3.9.11"
"3.9.10"
"3.9.9"
"3.9.8"
"3.9.7"
"3.9.6"
"3.9.5"
"3.9.4"
"3.9.3"
"3.9.2"
"3.9.2-rc.1"
"3.9.1"
"3.9.1-rc.1"
"3.9.0"
"3.9.0-rc.2"
"3.9.0-rc.1"
"3.9.0-beta.5"
"3.9.0-beta.4"
"3.8.14"
"3.8.13"
"3.8.12"
"3.8.11"
"3.8.10"
"3.8.9"
"3.8.8"
"3.8.7"
"3.8.6"
"3.8.5"
"3.8.4"
"3.8.3"
"3.8.2"
"3.8.1"
"3.8.0"
"3.7.15"
"3.7.14"
"3.7.13"
"3.7.12"
"3.7.11"
"3.7.10"
"3.7.9"
"3.7.8"
"3.7.7"
"3.7.6"
"3.7.5"
"3.7.1"
"3.6.15"
"3.6.14"
"3.6.13"
"3.6.12"
"3.6.11"
"3.6.10"
"3.6.9"
"3.6.8"
"3.6.7"
"3.5.10"
"3.5.9"
"3.5.4"
"2.7.18"
"2.7.17"
```

**Comment by JelleZijlstra** [comment]
Thanks for the analysis! I'd like to continue testing with the old .0 versions, so we should probably pin Ubuntu 20.04 for now.

**Comment by srittau** [comment]
Pinning a specific OS version in CI seems like a good idea anyway to prevent issues like this in the future. This gives us more time to properly test when a new OS version comes out.

**Comment by JelleZijlstra** [comment]
I tried to fix this and accidentally pushed straight to main: 5f9258d1efd5d8e4900ffc23ba2486240f2a2f35. To prevent this in the future I changed the repo setting to require PRs for main.

I merged main into #97 to see if the fix works, but GitHub seems to be laggy right now.

**Comment by AlexWaygood** [comment]
> I merged main into #97 to see if the fix works, but GitHub seems to be laggy right now.

CI seems to be hanging indefinitely on #97, possibly because GitHub is looking for a runner with `Ubuntu-2004` instead of `Ubuntu-20.04` :)

**Comment by AlexWaygood** [comment]
> > I merged main into #97 to see if the fix works, but GitHub seems to be laggy right now.
> 
> CI seems to be hanging indefinitely on #97, possibly because GitHub is looking for a runner with `Ubuntu-2004` instead of `Ubuntu-20.04` :)

#100 fixes it.
